tests/modeltests/tst_formatting.cpp

```diff
@@ -13,6 +13,8 @@
 
 #include "../testutils.h"
 
+#include <hotspot-config.h>
+
 #if KFSyntaxHighlighting_FOUND
```

Comment:
was this silently causing compiler warnings? warnings should ideally make the CI non-green (do we need -Werror?)

Reply:
oh no, it just didn't run the ksyntaxhighlighting tests

Reply:
I still don't understand this. it should have triggered a compiler warning before. why didn't it?